<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigConnection</class>
 <widget class="QDialog" name="ConfigConnection">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>216</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>381</width>
     <height>261</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>16</x>
       <y>64</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Tahoma</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>User:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>16</x>
       <y>86</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Tahoma</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Password:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edtLogFilePath">
     <property name="geometry">
      <rect>
       <x>126</x>
       <y>151</y>
       <width>241</width>
       <height>20</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>16</x>
       <y>108</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Tahoma</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Database:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edtSenha">
     <property name="geometry">
      <rect>
       <x>126</x>
       <y>82</y>
       <width>241</width>
       <height>20</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::PasswordEchoOnEdit</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>40</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Tahoma</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Server Name / IP:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>16</x>
       <y>153</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Tahoma</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Log File:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="sbPorta">
     <property name="geometry">
      <rect>
       <x>126</x>
       <y>127</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>50000</number>
     </property>
     <property name="value">
      <number>3306</number>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>10</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Tahoma</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Database Driver:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edtServer">
     <property name="geometry">
      <rect>
       <x>126</x>
       <y>37</y>
       <width>241</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>16</x>
       <y>132</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Tahoma</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Port:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbDriver">
     <property name="geometry">
      <rect>
       <x>126</x>
       <y>8</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>MySQL </string>
      </property>
      <property name="icon">
       <iconset resource="../resources/Resources.qrc">
        <normaloff>:/icons/1444503647_mysql-2048-black.png</normaloff>:/icons/1444503647_mysql-2048-black.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PostgreSQL</string>
      </property>
      <property name="icon">
       <iconset resource="../resources/Resources.qrc">
        <normaloff>:/icons/postgresql-96x96.png</normaloff>:/icons/postgresql-96x96.png</iconset>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="edtUsuario">
     <property name="geometry">
      <rect>
       <x>126</x>
       <y>60</y>
       <width>241</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="edtDatabase">
     <property name="geometry">
      <rect>
       <x>126</x>
       <y>105</y>
       <width>241</width>
       <height>20</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>207</x>
       <y>180</y>
       <width>158</width>
       <height>26</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnOk">
        <property name="text">
         <string>Migrate</string>
        </property>
        <property name="icon">
         <iconset resource="../resources/Resources.qrc">
          <normaloff>:/icons/Ok.png</normaloff>:/icons/Ok.png</iconset>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCancel">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
        <property name="icon">
         <iconset resource="../resources/Resources.qrc">
          <normaloff>:/icons/Cancel.png</normaloff>:/icons/Cancel.png</iconset>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>361</width>
       <height>201</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Syncing database schema</string>
          </property>
          <property name="textFormat">
           <enum>Qt::RichText</enum>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextBrowser" name="memoLog"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="lbStatus">
        <property name="text">
         <string>Status</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../resources/Resources.qrc"/>
 </resources>
 <connections/>
</ui>
